ci/dep-linux.sh

```diff
@@ -4,57 +4,107 @@
 set -ev
 
 # Install packages
+echo " IN DEP SCRIPT "
+env
+pwd
+ls -ltr
+echo ~
+ls -ltr ~
 
-case "$CXX" in
+# Verify CMake 3
+cmake --version
+
+# Install dependencies (libxc, mpich)
+
+export FC=/usr/bin/gfortran-8
+
+# Set up CC and CXX commands
+export COMPILER=$CXX
+case "$COMPILER" in
     g++)
-        export CC=/usr/bin/gcc-$GCC_VERSION
-        export CXX=/usr/bin/g++-$GCC_VERSION
+        echo "setting up for GCC version 8"
+        export CC=/usr/bin/gcc-8
+        export CXX=/usr/bin/g++-8
         ;;
     clang++)
-        export CC=/usr/bin/clang-3.8
-        export CXX=/usr/bin/clang++-3.8
+        echo "setting up for CLANG version 7"
+
+        which clang || true
+        which clang++ || true
+        which clang++-7.0 || true
+        export CC=clang
+        export CXX=clang++
         ;;
     *)
         echo "Unknown C++ compiler:"
         echo "$CXX"
         exit 1
         ;;
 esac
-export FC=/usr/bin/gfortran-$GCC_VERSION
 
-# Print compiler information
+# Confirm compiler information
 $CC --version
 $CXX --version
 $FC --version
 
+
+# Install ccache
+if [ ! -f "${HOME}/ccache/bin/ccache" ]; then
```

Comment:
Need the latest version of ccache to work around issues with command line arguments breaking the caching.  With latest version we are still only caching about 30% of the compilations and unfortunately the most expensive ones are not being cached (mra[1-6].o) --- am still trying to diagnose the root cause since on my laptop with the same version software I get 100% caching.